model/Licensing/Classes/License.md

```diff
@@ -14,30 +14,11 @@ A License represents a license text, whether listed on the SPDX License List
 ## Metadata
 
 - name: License
-- SubclassOf: AnyLicenseInfo
+- SubclassOf: LicensingText
```

Comment:
In SPDX 2.X this is called "SimpleLicensingInfo" - a bit long, but I would prefer the existing name.